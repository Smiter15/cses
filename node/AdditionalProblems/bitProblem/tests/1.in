100
62 54 37 59 17 83 39 79 93 31 100 21 66 92 11 83 40 7 70 35 41 81 81 67 84 60 47 96 80 32 70 92 46 27 21 7 53 62 58 5 41 66 62 68 64 20 29 46 92 80 32 80 62 44 80 55 78 56 65 85 22 98 53 53 78 6 50 74 46 90 81 49 76 73 36 56 57 26 7 14 6 18 82 20 96 84 88 56 43 61 89 94 43 75 16 64 35 7 36 68
