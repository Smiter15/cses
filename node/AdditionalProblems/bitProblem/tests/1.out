29 4 97
13 6 96
6 8 85
23 1 96
2 22 72
19 2 94
15 3 92
26 1 97
31 1 98
21 1 94
11 1 94
7 11 89
4 16 73
18 5 96
1 9 80
19 2 94
3 22 62
7 9 84
10 6 88
4 8 82
5 8 78
12 8 88
12 8 88
6 5 87
13 8 90
13 6 94
27 1 93
6 4 79
7 21 73
2 41 41
10 6 88
18 5 96
12 8 88
6 3 89
7 11 89
7 9 84
16 5 95
29 4 97
11 6 94
1 20 74
5 8 78
4 16 73
29 4 97
4 17 77
2 42 42
3 26 78
8 4 91
12 8 88
18 5 96
7 21 73
2 41 41
7 21 73
29 4 97
6 11 81
7 21 73
30 1 98
15 4 95
7 12 83
3 13 71
22 2 97
7 9 89
9 1 92
16 5 95
16 5 95
15 4 95
2 26 73
5 9 92
5 7 84
12 8 88
15 2 94
12 8 88
5 8 85
5 9 91
4 5 87
4 21 72
7 12 83
12 3 90
3 11 86
7 9 84
3 14 83
2 26 73
2 19 80
11 5 90
3 26 78
6 4 79
13 8 90
8 8 85
7 12 83
10 4 87
27 1 96
15 2 94
34 1 98
10 4 87
10 2 93
1 52 52
2 42 42
4 8 82
7 9 84
4 21 72
4 17 77
